Backend/routes/listings.js

```diff
@@ -137,7 +135,7 @@ router.get("/listings/:id", logActivity(), async (req, res) => {
   }
 });
 
-router.delete("/listings/:id", verifyToken, async (req, res) => {
+router.delete("/listings/:id", verifyToken, verifyToken, async (req, res) => {
```

Comment:
did you mean to add logActivity instead of verifyToken twice here?

Reply:
yes thats right